src/LessMsi.Gui/MainForm.cs

```diff
@@ -237,23 +240,23 @@ public void AddPropertyGridColumn(string boundPropertyName, string headerText)
 		public ComboBox cboTable;
 		private Label label2;
 		private Panel panel1;
-		private Button btnExtract;
+		public Button btnExtract;

```

Comment:
why did several of these buttons become public? These should almost certainly be private.

Reply:
Aaahhh. I see it was for the test now.